gui/doc/indicator.md

```diff
@@ -51,11 +51,33 @@ The _orientation_ can be specified to "vertical" (or "v") to create a vertical i
     === "Markdown"
 
         ```
-        <|{50}|indicator|orientation=v|value=10|>
+        <|message|indicator|orientation=v|value=10|>
```

Comment:
Could this be more explicit?

Like, KPI?

Reply:
I thought "message" was nicer that {50} but if you have a better idea ... ;-)